<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmExplore</class>
 <widget class="QDialog" name="FrmExplore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Explorador</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(125, 135, 147, 255), stop:0.982955 rgba(201, 206, 212, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnAtras">
        <property name="toolTip">
         <string>Atr√°s</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
     border: 1px solid #000000;

     
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

     min-width: 30px;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {
   
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));

 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    </string>
        </property>
        <property name="icon">
         <iconset resource="../radit.qrc">
          <normaloff>:/general/iconos/menu/flecha.png</normaloff>:/general/iconos/menu/flecha.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BtnPlay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Reproducir por monitor</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
     border: 1px solid #000000;

     
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

     min-width: 30px;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {
   
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));

 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../radit.qrc">
          <normaloff>:/general/iconos/player/play.png</normaloff>:/general/iconos/player/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnPause">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Pause</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
     border: 1px solid #000000;

     
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

     min-width: 30px;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {
   background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../radit.qrc">
          <normaloff>:/general/iconos/player/pause.png</normaloff>:/general/iconos/player/pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnStop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
     border: 1px solid #000000;

     
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

     min-width: 30px;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {    
background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));

 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../radit.qrc">
          <normaloff>:/general/iconos/player/stop.png</normaloff>:/general/iconos/player/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LinePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnRed">
        <property name="styleSheet">
         <string notr="true">QPushButton {
   
	color: rgb(255, 255, 255);
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {
   
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));

 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    
    </string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineSeek">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnSeek">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Buscar</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
     border: 1px solid #000000;

     
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

     min-width: 30px;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {    
background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));

 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../radit.qrc">
          <normaloff>:/general/iconos/menu/lupa.png</normaloff>:/general/iconos/menu/lupa.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnSalir">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Cerrar</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
   
	color: rgb(255, 255, 255);
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(67, 73, 85, 255), stop:0.982955 rgba(143, 150, 168, 255));

 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

QPushButton:hover {
   
	background-color: qlineargradient(spread:pad, x1:0.994318, y1:1, x2:1, y2:0, stop:0.0113636 rgba(69, 142, 187, 255), stop:0.982955 rgba(95, 179, 241, 255));

 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
    </string>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="TableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header location="global">explore/TableView/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../radit.qrc"/>
 </resources>
 <connections/>
</ui>
